README.md

```diff
@@ -661,4 +665,4 @@ Feel free to ask for help [on the discussions page](https://github.com/tgstation
 * The /tg/station 13 icon is licensed under [Creative Commons 3.0 BY-SA](http://creativecommons.org/licenses/by-sa/3.0/).
 * The remainder of the project is licensed under [GNU AGPL v3](http://www.gnu.org/licenses/agpl-3.0.html)
 
-See the files in the `/src/DMAPI` tree for the MIT license
+See the files in the `/src/DMAPI` tree for the MIT license
```

Comment:
Last one is a missing new line.

Reply:
Does this mean that line 669 should be blank?

Reply:
Yes

Reply:
All done! Thank you for your patience and responsiveness Cyberboss :)